docs/critical-header-metadata-warning.md

```
@@ -0,0 +1,224 @@
+```
+
+Or footers like:
+```yaml
+## Version Notes
+```yaml
+pi_version: v1.0
+pi_date: 2025-09-01
+approach: pure_advisor_personality
+player_context: none
+```
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)